index.html

```diff
@@ -6,43 +6,178 @@
+            </li>
+            <li>
+              <a href=""> <img src="./assets/linkedin.png" alt="linkedin" /></a>
+            </li>
+            <li>
+              <a href=""><img src="./assets//Vector.png" alt="vector1" /></a>
+            </li>
+            <li>
+              <a href=""><img src="./assets//twitter.png" alt="twitter" /></a>
+            </li>
+            <li>
+              <a href=""><img src="./assets//Vector2.png " alt="vector2" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="works">
+        <div class="title">
+          <span>My Recent Works</span>
+          <hr />
+        </div>
+        <div class="work-container">
+          <div class="hero-work">
+            <img src="./assets//hero-work.png" alt="hero" />
+            <span>Multi-Post Stories</span>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">css</div>
+              </li>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
+          </div>
+          <div class="work-card">
+            <div class="title">Profesional Art Printing Data</div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
+          </div>
+          <div class="work-card">
+            <div class="title">Profesional Art Printing Data</div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
```

Comment:
- [ ] [REQUIRED] Replace this with a `button` or an `a `tag. refer to my previous comment on this.